partner-chains-cli/src/setup_main_chain_state/mod.rs

```diff
@@ -94,6 +96,11 @@ impl CmdRun for SetupMainChainStateCmd {
 		context.print(
 			"This wizard will set or update D-Parameter and Permissioned Candidates on the main chain. Setting either of these costs ADA!",
 		);
+		if !context.file_exists(SIDECHAIN_MAIN_CLI_PATH) {
+			return Err(anyhow!(
+				"Partner Chains CLI executable file ({SIDECHAIN_MAIN_CLI_PATH}) is missing."
```

Comment:
Partner Chains CLI is partner-chains-cli. What is missing is smart contracts CLI. This message is confusing.

Reply:
fixed that